src/Nowruz/modules/Auth/containers/forgetPassword/EmailForm/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { BackLink } from 'src/Nowruz/modules/general/components/BackLink';
+import { Button } from 'src/Nowruz/modules/general/components/Button';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+
+import { useEmailForm } from './useEmailForm';
+
+export const EmailForm = () => {
+  const { register, errors, navigateToOtp, handleSubmit, onBack } = useEmailForm();
+
+  return (
+    <>
+      <form className="mt-[32px] mb-[32px]">
```

Comment:
mt-5 is 32px

Reply:
![image](https://github.com/socious-io/web-app-v2/assets/60564928/67069e43-8517-4c0c-9abd-ac4508aedbad)